c_src/enif_protobuf.h

```diff
@@ -41,31 +41,32 @@
 
 #if IS_DEBUG
 #define debug_out(fmt) do { \
-    fprintf(stderr, "%s:%d:%s(): " fmt "\r\n", __FILE__, __LINE__, __func__); \
+    printf("%s:%d:%s(): " fmt "\r\n", __FILE__, __LINE__, __func__); \
```

Comment:
I kind of like these debuggers printing to stderr given that it's more reliably flushed. Personal preference but I would prefer keeping these as `fprintf(stderr...`